### PR TITLE
Guidance: Expand main explainer text about CrateDB Cloud

### PR DESCRIPTION
**Pull request by amotl**
## What's Inside
Just a few ideas about the »**About CrateDB Cloud**« topic in general, to better convey **What's Inside**, intending to provide **better guidance**.

## Preview
https://crate-cloud--58.org.readthedocs.build/en/58/

## Highlights
Try to add some fresh air, with phrases from others doing similar business. They can't be that wrong?

## Thoughts

If you like the idea, please support us shaping it by adding your comments and suggestions, or approving. Thanks!

~~Not all details need to go into the final version, and alternatively, the PR might be closed without further ado if there is no interest or capacities to improve this topmost section of the cloud documentation.~~

~~Another possibility is that the content will go elsewhere, but our intention was to conceive it for the topmost explainer area. In the end, everything is possible, as we just wanted to submit it for discussion. Maybe you can come up with better wording if you are up to it, or you will merge it without further ado ;], and re-shape it on subsequent iterations. Thanks!~~

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,11 +3,35 @@
...
-back up, and scale your [CrateDB] clusters in the cloud without the need to do
-database management yourself. You can store and analyze massive amounts of data
-in near real-time, even with complex queries.
+CrateDB Cloud is a serverless and cost-effective enterprise data warehouse
+SQL database that works across clouds and scales with your data.
+Use built-in ML/AI and BI for insights at scale.
 
+CrateDB Cloud is a fully managed, petabyte-scale, and cost-effective analytics
+database that lets you run analytics over vast amounts of data in near real
+time.
```
Comment (amotl) [inline docs/index.md:12]: Not sure if petabyte-scale is applicable, or if we better should scale down to terabytes?

File: docs/index.md [hunk 2]
```diff
@@ -3,11 +3,35 @@
...
+CrateDB Cloud is a fully managed enterprise service, allowing you to deploy,
+monitor, back up, and scale your CrateDB clusters in the cloud without the
+need to do database management yourself. You can store and analyze massive
+amounts of data in near real time, even with complex queries.
+
+:::{rubric} Learn
+:::
 Users around the world rely on CrateDB Cloud clusters to store billions of records
 and terabytes of data, all accessible without delays. If you want to start using
 CrateDB Cloud, or make the most of your existing subscription, we are maintaining
```
Comment (amotl) [inline docs/index.md:37]: Just some, potentially incomplete ideas about expanding the single-paragraph explainer text »About CrateDB Cloud« into something with more details about "What's Inside". If you like the idea in general, please support us shaping it by adding your comments and suggestions. Thanks!

File: docs/index.md [hunk 3]
```diff
@@ -3,11 +3,35 @@
...
+:::{rubric} Operational Features
+:::
+With CrateDB Cloud, there's no infrastructure to set up or manage, letting you
+focus on finding meaningful insights using plain SQL, and taking advantage of
+flexible pricing models across on-demand and flat-rate options.
+
+CrateDB Cloud is a fully managed enterprise service, allowing you to deploy,
+monitor, back up, and scale your CrateDB clusters in the cloud without the
+need to do database management yourself. You can store and analyze massive
+amounts of data in near real time, even with complex queries.
```
Comment (amotl) [inline docs/index.md:31]: Not strictly "operational", so it's in the wrong section ("Operational Features") when being overly nit-picky. However, it was the original explainer, which in itself, hasn't been refactored, just moved.

Right now, the outcome is still balanced, so we think the sentence is still a sensible way to conclude the "Operational" miniature subsection, so we'd keep it for now. Tomorrow, we will probably think about it in a totally different way, and apply further refactorings to it.

Comment (amotl) [reply docs/index.md]: Refactored ;].

File: docs/index.md (hunk 1 above)
Comment (amotl) [reply docs/index.md]: @seut mentioned CrateDB has been validated on petabyte-scale successfully.

File: docs/index.md [hunk 4]
```diff
@@ -3,11 +3,54 @@
 
 # CrateDB Cloud
 
-**CrateDB Cloud** is a fully-managed service, allowing you to deploy, monitor,
-back up, and scale your [CrateDB] clusters in the cloud without the need to do
-database management yourself. You can store and analyze massive amounts of data
-in near real-time, even with complex queries.
+CrateDB Cloud is a fully managed, petabyte-scale, and cost-effective
```
Comment (proddata) [inline docs/index.md:6]:
```suggestion
CrateDB Cloud is a fully managed and cost-effective
```

File: docs/index.md [hunk 5]
```diff
@@ -3,11 +3,54 @@
...
+analytics database that lets you run analytics over vast amounts of
+data in near real time, even with complex queries.
 
+It is an SQL database service for enterprise data warehouse workloads,
+that works across clouds and scales with your data. Use built-in ML/AI
```
Comment (proddata) [inline docs/index.md:11]: What built-in ML/AI are you referring to?

File: docs/index.md [hunk 6]
```diff
@@ -3,11 +3,54 @@
...
+:padding: 0
+
+::::{grid-item}
+:class: rubric-slimmer
+:columns: 6
+
+:::{rubric} Database Features
+:::
+CrateDB Cloud helps you manage and analyze your data with built-in features
+like machine learning, geospatial analysis, and business intelligence.
```
Comment (proddata) [inline docs/index.md:25]: what machine learning is built-in?

File: docs/index.md [hunk 7]
```diff
@@ -3,11 +3,54 @@
...
+
+CrateDB Cloud's scalable, distributed analysis engine lets you query terabytes
+in seconds and petabytes in minutes.
```
Comment (proddata) [inline docs/index.md:28]: Also petabytes could in theory be queried in milliseconds. I think this is misleading

File: docs/index.md (hunk 4 above)
Comment (amotl) [reply docs/index.md]: 30943fb33fc adjusts the text, also addressing all the other spots you had remarks on.

File: docs/index.md (hunk 5 above)
Comment (amotl) [reply docs/index.md]: Removed. Thanks.

File: docs/index.md (hunk 6 above)
Comment (amotl) [reply docs/index.md:24]: Removed.

File: docs/index.md (hunk 7 above)
Comment (amotl) [reply docs/index.md]: Adjusted.

**Review by proddata — APPROVED** [review]
thanks 👍

**Pull request merged**